app/src/main/java/arjun/hanabiandroid/game/InitialState.java

```diff
@@ -0,0 +1,38 @@
+package arjun.hanabiandroid.game;
+
+import android.support.annotation.IntDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+public class InitialState {
+    @Retention(RetentionPolicy.SOURCE)
+    @IntDef({
+            GAME_TYPE_NO_RAINBOW,
+            GAME_TYPE_RAINBOW_SUIT,
+            GAME_TYPE_RAINBOW_WILDCARD,
+    })
+    @interface GameType {
+    }
```

Comment:
So this isn actually just ints, with some compile time type safety. Using enums is a little expensive, so it's better to avoid them unless you need methods on the enum values (as in `Color`)